.circleci/config.yml

```diff
@@ -109,6 +110,7 @@ jobs:
                     pip install -r dev-requirements.txt --quiet
                     git clone --depth 1 git@github.com:plotly/dash.git dash-main
                     pip install -e ./dash-main --quiet
+                    cd dash-main && pip install -r requires-ci.txt
```

Comment:
does this not work as `pip install -e ./dash-main[ci]`?

Reply:
should work. updating